modules/core/shared/src/main/scala/lucuma/core/model/sequence/DynamicConfig.scala

```diff
@@ -6,18 +6,17 @@ package lucuma.core.model.sequence
 import cats.Eq
 import cats.syntax.all._
 import lucuma.core.enums._
+import lucuma.core.util.TimeSpan
 import monocle.Focus
 import monocle.Lens
 import monocle.Prism
 import monocle.macros.GenPrism
 import org.typelevel.cats.time._
 
-import java.time.Duration
-
 sealed trait DynamicConfig
 object DynamicConfig {
   final case class GmosNorth(
-    exposure:      Duration,
+    exposure:      TimeSpan,
```

Comment:
Previously we allowed negative `Duration` but with `TimeSpan` it must be non-negative.  I think we weren't intending to use negative exposure times for anything since it doesn't correspond to reality.

Reply:
is a 0 exposure time allowed?

Reply:
yeah, biases for example are 0 seconds

Reply:
I've never understood that, especially that when I do a bias in the real world they do take time 😄

Reply:
Ha, yes it still has to configure and readout presumably even if there is no exposure time.